Sdkmigration

CryptoAssassin1 opened the pull request.

CryptoAssassin1 committed:
sdkmerge

CryptoAssassin1 committed:
Merge upstream/sdkmigration: resolve modify/delete by keeping ours

CryptoAssassin1 committed:
Merge pull request #91 from CryptoAssassin1/sdkmigration-port

SDK migration merge

CryptoAssassin1 committed:
Add new docs

CryptoAssassin1 committed:
fixed m

CryptoAssassin1 committed:
fixed mod

CryptoAssassin1 committed:
v0.53 Modules fixed

CryptoAssassin1 committed:
v0.50 Modules fixed

CryptoAssassin1 committed:
authre

CryptoAssassin1 committed:
module

CryptoAssassin1 committed:
fixed landing page links

CryptoAssassin1 committed:
fixed Building apps

CryptoAssassin1 committed:
fixed Building apps v0.53

CryptoAssassin1 committed:
v0.53 module landing

CryptoAssassin1 committed:
v0.47 Modules fixed

CryptoAssassin1 committed:
v0.47 module

CryptoAssassin1 committed:
update docs.json

CryptoAssassin1 committed:
module .47

CryptoAssassin1 committed:
module .50 replaced with .47

CryptoAssassin1 committed:
list of mod

CryptoAssassin1 committed:
Building app nav .47

CryptoAssassin1 committed:
remove autocli

CryptoAssassin1 committed:
glossary page added

CryptoAssassin1 committed:
adr added

CryptoAssassin1 committed:
adrs path update

CryptoAssassin1 committed:
adrs added v0.50

CryptoAssassin1 committed:
adrs update v0.50

CryptoAssassin1 committed:
.47 tutorial tab removed

CryptoAssassin1 committed:
overview app

CryptoAssassin1 committed:
adrs added v0.53

CryptoAssassin1 committed:
paths fixed .53

CryptoAssassin1 committed:
store and fix

CryptoAssassin1 committed:
store file rename

CryptoAssassin1 committed:
ics added

CryptoAssassin1 committed:
tooling sec update

CryptoAssassin1 committed:
final .50fix

CryptoAssassin1 committed:
all warnings cleared

CryptoAssassin1 committed:
removed extra user group

CryptoAssassin1 committed:
next folder n rmve extra user group

CryptoAssassin1 committed:
images update

CryptoAssassin1 committed:
images update

CryptoAssassin1 committed:
path fixes

CryptoAssassin1 committed:
app anotmy path

CryptoAssassin1 committed:
add landing page

CryptoAssassin1 committed:
tx lifecyle path update

CryptoAssassin1 committed:
tx lifecyle path

CryptoAssassin1 committed:
quer lifecycle

CryptoAssassin1 committed:
cli update

Cordtus commented:
I think just change this top one to "Cosmos SDK iin each tab" 
<img width="748" height="492" alt="image" src="https://github.com/user-attachments/assets/efb283dd-22c1-45cb-8d48-a052ee9578e2" />

CryptoAssassin1 committed:
bech32

CryptoAssassin1 committed:
bech32 up

CryptoAssassin1 committed:
app secific

CryptoAssassin1 committed:
inter block

CryptoAssassin1 committed:
inter block .47

CryptoAssassin1 committed:
store update

CryptoAssassin1 committed:
learn architecture

CryptoAssassin1 committed:
update readme

CryptoAssassin1 committed:
update buf

Cordtus committed:
Update docs.json

Change top-level group labels in each dropdown/version/tab
Remove icons from each dropdown

Cordtus committed:
Update docs.json

Fix 'SDKs' >> 'SDK'

Cordtus commented:
Looks good
